### ISSUE TITLE
Prep project boards for Migration

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/ops#131]
### Overview
We need to prepare the Project Boards for the forced migration that GitHub is doing in August, so that it is already working well for the team by then.

### Action Items
- [ ]  copy this template to a comment below
   ````
   ### Text from project board card
   ```
   [INSERT MARKDOWN HERE]
   ```
   - [ ] Determine where this item goes on the wiki (page and section)
   - [ ] copy item to wiki
   - [ ] hide this comment when completed.
   ````
- [ ] copy markdown from a card in the "Start Here" column of the Project Board into your new comment where it says `[INSERT MARKDOWN HERE]
- [ ] delete card from project board.

### Resources/Instructions
Project board URL:

**Comment by ExperimentsInHonesty** [comment]
### Text from project board card,  New Issue Review column
```
All issues that are newly created or coming out of the icebox need to be reviewed by Product to make sure they are complete and prioritized
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by ExperimentsInHonesty** [comment]
### Text from project board card
```
Prioritized Backlog = features that are ready to work and a priority. This column should be prioritized by most important first.
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by ExperimentsInHonesty** [comment]
### Text from project board card, In Progress
```
In Progress = tasks that have been assigned to team members and are being actively worked on. 
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by ExperimentsInHonesty** [comment]
### Text from project board card
```
Questions/Review =  Issues that you are blocked on because you need an answer from another team member or other type of question/review.
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by ExperimentsInHonesty** [comment]
### Text from project board card, Links/Recurring Tasks
```
Links = Cards that have links that you need to get to often.
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by ExperimentsInHonesty** [comment]
### Text from project board card, Start Here
```
Start Here = Includes things like meeting times and zoom links, and anything the team members need to read.
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by ExperimentsInHonesty** [comment]
### Text from project board card
```
[DevOps Wiki](https://github.com/hackforla/ops/wiki)
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by ExperimentsInHonesty** [comment]
### Text from project board card
```
Meeting Times
- Wednesday's at 6PM Pacific

_To get onto the calendar invites, send your email with your request to one of the [Ops CoP leads](https://github.com/hackforla/ops/wiki/Community#ops-community-of-practice-cop-leads) on Slack._
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by ExperimentsInHonesty** [comment]
A link to https://github.com/hackforla/ops/issues/100

**Comment by ExperimentsInHonesty** [comment]
### Text from project board card, Open Roles, Instructions
```
# START HERE
We know this board is ugly!!!

- [Find a volunteer Opportunity](https://github.com/hackforla/ops/projects/1#card-53859707)
- [Post a volunteer Opportunity](https://github.com/hackforla/ops/projects/1#card-53859697)
- [Remove a volunteer Opportunity](https://github.com/hackforla/ops/projects/1#card-53859680)

If you know some places where we should post these opportunities please [submit Ops channels](https://docs.google.com/forms/d/e/1FAIpQLSdJFYAa6mt2hA4NskFL85_p00ZHT8ThIECdO1wpyFGx0Xp3DA/viewform) after you check to make sure its not on the list of [existing Development channels we know about](https://docs.google.com/spreadsheets/d/1sPu4ylVz20SHYSezoo_DDO_gh1ifv0VE2QlCoFwRaRA/preview)
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by ExperimentsInHonesty** [comment]
### Text from project board card, Open Roles, Instructions
```
#### Find a volunteer opportunity
---

**Steps**
1. [Signup for the Hack for LA Slack Workspace](http://hackforla.org/slack)
1. If you don't already have a GitHub account, please [signup](https://github.com).  We use boards like this to organize information quickly.
1. Read the role descriptions at the top of the columns, pick which is appropriate for your existing skill level
1. Read through the opportunities posted by the teams, pick one and communicate with the team by following the instructions on the opportunity cards.
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by ExperimentsInHonesty** [comment]
### Text from project board card, Open Roles, Instructions
```
#### Post a Volunteer Opportunity
---
Steps:
1. Get added to this board (reach out to Olivia or Bonnie to be added)
1. click on the plus sign at the top of the column to start a new card under the role type your project falls under
1. If the role type dosen't exist, please make a new column and add a description ([they can be found here](https://github.com/hackforla/civic-opportunity/tree/master/roles-1))
1. Use the following format:

**Project Name**: [your project name replaces these brackets]

**Volunteer Opportunity**: [your opportunity description replaces these brackets]

**Duration**: 

**Who to communicate your interest to**
- Slack channel link: #
- Slack name of person to contact in the channel

[Read more about the project]([your project home page on hackforla.org website goes here])

**Resources:**
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by ExperimentsInHonesty** [comment]
### Text from project board card, Open Roles, Instructions
```
#### Remove a Volunteer Opportunity
---

**Steps**
1. Confirm that you are the UX or PM lead for your project and that your PM(s) wants this opportunity removed.
1. Edit the opportunity by click the three dots ... on the top right corner of the card.
1. Add  ~~ at the top and the bottom of the card (eg. ~~strike this text out~~)
1. Move to the filled column
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by ExperimentsInHonesty** [comment]
Text from project board card, Open Roles, Filled

### Text from project board card
```
No projects are currently looking for this role.  If you are interested in helping us build out this capability, please join us at our weekly community of practice meeting.  We are in the process of building documentation and could use your help.
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by ExperimentsInHonesty** [comment]
1. Progress: We did the urgent stuff and I will come back to the rest
2. Blockers: Rabia and I are busy right now
3. Availability: none for the next two weeks
4. ETA: 3 weeks

* remember to add links to the top of the issue if they are going to be needed again.

**Comment by ExperimentsInHonesty** [comment]
Done!
